addons/account/tests/test_invoice_taxes.py

```diff
@@ -164,9 +165,10 @@ def test_group_of_taxes(self):
             (100, self.percent_tax_2),
         ])
         invoice.action_post()
-        self.assertRecordValues(invoice.line_ids.filtered('tax_line_id').sorted(lambda x: x.price_unit), [
-            {'name': self.percent_tax_1_incl.name,      'tax_base_amount': 100, 'price_unit': 21,      'tax_ids': [self.percent_tax_2.id]},
-            {'name': self.percent_tax_2.name,           'tax_base_amount': 221, 'price_unit': 26.52,   'tax_ids': []},
+        self.assertRecordValues(invoice.line_ids.filtered('tax_line_id').sorted('balance'), [
+            {'name': self.percent_tax_1_incl.name,      'tax_base_amount': 100.0,   'balance': -21.0,   'tax_ids': [self.percent_tax_2.id]},
+            {'name': self.percent_tax_2.name,           'tax_base_amount': 121.0,   'balance': -14.52,  'tax_ids': []},
+            {'name': self.percent_tax_2.name,           'tax_base_amount': 100.0,   'balance': -12.0,   'tax_ids': []},
```

Comment:
why is it not grouped anymore in the journal items?

Reply:
Because `percent_tax_2` is contained by `group_tax`. Since I added a field tracking the group of taxes, this is creating two lines for `percent_tax_2`: one having `tax_group_id=group_tax.id`, other having `tax_group_id=False`